config.example.wikimedia.yaml

```diff
@@ -449,6 +442,168 @@ templates:
             - get_from_backend:
                 request:
                   uri: https://wikimedia.org/api/rest_v1/metrics/pageviews/top/{+rest}
+      /{module:graphoid}/v1/store/{format}:
```

Comment:
An empty line between the pageviews and graphoid modules would increase readability.